### PR TITLE
Fix #184349 report.az

### LINKED ISSUE #184349: report.az
### Issue URL (Annoyance)
 [https://report.az/ru/v-regione/osuzhdennyj-za-ubijstvo-zheny-eks-ministr-kazahstana-bishimbaev-otpravlen-v-koloniyu/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Freport.az%2Fru%2Fv-regione%2Fosuzhdennyj-za-ubijstvo-zheny-eks-ministr-kazahstana-bishimbaev-otpravlen-v-koloniyu%2F)
Username: `@zubrRB`

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2 </summary>

  Screenshot 2
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.18.1 (4778)
Browser: | Firefox
License type: | paid
AdGuard driver: | TDI
Stealth mode options: | Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties (example.com), <br>Hide your User-Agent, <br>Hide your IP address, <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (1), <br>Disable cache for third-party requests, <br>Block trackers, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `System`<br>filters: `https://filters.adtidy.org/windows/filters/15.txt`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Ukrainian
Userscripts: | https://github.com/Owyn/HandyImage/raw/master/HandyImage.user.js<br>https://update.greasyfork.org/scripts/14085/kinozaltv magnet links.user.js<br>https://update.greasyfork.org/scripts/394820/Mouseover Popup Image Viewer.user.js<br>https://download.sf-helper.com/chrome/helper.user.js<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js<br>https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js

### Import issue configuration (AdGuard for Windows)
[Import](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=adguard%3Aimport_user_configuration%3Fbrowsing_security.enabled%3Dtrue%26browsing_security.statistics_enabled%3Dfalse%26dns.custom_bootstrap%3D%26dns.custom_fallback%3D%26dns.enabled%3Dtrue%26dns.fallback_mode%3DSystem%26dns.filters%3Dhttps%253A%252F%252Ffilters.adtidy.org%252Fwindows%252Ffilters%252F15.txt%26dns.filters_enabled%3Dtrue%26dns.servers%3DSystem%26dns.timeout%3D5000%26filters%3D14.17.1.23.2.3.4%26license_type%3Dpaid%26parental_control.enabled%3Dfalse%26product_type%3DWin%26product_version%3D7.18.1%2B%25284778%2529%26referrer%3D%26stealth.DNT%3Dtrue%26stealth.block_third_party_auth%3Dfalse%26stealth.block_trackers%3Dtrue%26stealth.disable_third_party_cache%3Dtrue%26stealth.disable_wap_push_message_routing_service%3Dfalse%26stealth.disable_windows_defender%3Dfalse%26stealth.disable_windows_telemetry%3Dtrue%26stealth.dpi%3Dtrue%26stealth.enabled%3Dtrue%26stealth.flash%3Dfalse%26stealth.hide_search_queries%3Dtrue%26stealth.ip%3D%26stealth.java%3Dfalse%26stealth.location%3Dtrue%26stealth.push%3Dtrue%26stealth.referrer%3Dexample.com%26stealth.strip_url%3Dtrue%26stealth.third_party_cookies%3D1%26stealth.turn_off_advertising_id%3Dtrue%26stealth.user_agent%3D%26stealth.webrtc%3Dtrue%26stealth.x_client%3Dtrue%26system_version%3DMicrosoft%2BWindows%2BNT%2B6.1.7601%2BService%2BPack%2B1%26url%3Dhttps%253A%252F%252Freport.az%252Fru%252Fv-regione%252Fosuzhdennyj-za-ubijstvo-zheny-eks-ministr-kazahstana-bishimbaev-otpravlen-v-koloniyu%252F%26user_agent%3DMozilla%252F5.0%2B%2528Windows%2BNT%2B6.1%253B%2BWin64%253B%2Bx64%253B%2Brv%253A109.0%2529%2BGecko%252F20100101%2BFirefox%252F115.0%26userscripts%3Dhttps%253A%252F%252Fgithub.com%252FOwyn%252FHandyImage%252Fraw%252Fmaster%252FHandyImage.user.js%252Chttps%253A%252F%252Fupdate.greasyfork.org%252Fscripts%252F14085%252Fkinozaltv%252520magnet%252520links.user.js%252Chttps%253A%252F%252Fupdate.greasyfork.org%252Fscripts%252F394820%252FMouseover%252520Popup%252520Image%252520Viewer.user.js%252Chttps%253A%252F%252Fdownload.sf-helper.com%252Fchrome%252Fhelper.user.js%252Chttps%253A%252F%252Fuserscripts.adtidy.org%252Frelease%252Fadguard-extra%252F1.0%252Fadguard-extra.user.js%252Chttps%253A%252F%252Fuserscripts.adtidy.org%252Frelease%252Fpopup-blocker%252F2.5%252Fpopupblocker.user.js%26win.wfp%3Dfalse)

### PR DESCRIPTION
**Pull request by Chinaski1**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [ ] This is not an ad/bug report;
- [ ] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [ ] I have performed a self-review of my own changes;
- [ ] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Example: <https://github.com/AdguardTeam/AdguardFilters/issues/184349>

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

</details>

<details>

<summary>Screenshot N:</summary>

</details>

### Terms

- [ ] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Other/sections/annoyances.txt [hunk 1]
```diff
@@ -3229,3 +3229,6 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 !+ PLATFORM(ios) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app#$#.hello_from_adguard_advanced_protection_ios { display: none !important; }
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184349
+report.az##main.content:nth-child(3) > section.news-inner-page.pt-20:first-child > div.container > div.row > div.col-lg-8.news-col:first-child > div.news-wrapper > div.selected-news:first-child > div.editor-body:nth-child(8) > div.subs-in-social.subs-telegram:last-child
+report.az##main.content:nth-child(3) > section.news-inner-page.pt-20:first-child > div.container > div.row > div.col-lg-8.news-col:first-child > div.news-wrapper > div.selected-news:first-child > div.editor-body:nth-child(8) > div.subs-in-social.subs-instagram:last-child
```
Comment (piquark6046) [inline AnnoyancesFilter/Other/sections/annoyances.txt:3234]: We don't usually use a rule like this formation.
Please take a look the repo softly.

Comment (Alex-302) [inline AnnoyancesFilter/Other/sections/annoyances.txt:3234]: Правила, сделанные помощником не котируются. Надо изучать страницу и код. Там видно, как сделать качественное правило.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by piquark6046 — CHANGES_REQUESTED** [review]
I think that they can be merged to `report.az##div.subs-in-social`.

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt [hunk 2]
```diff
@@ -7076,3 +7076,6 @@ magazine.skyeng.ru##.tg-banner
 ! NOTE: Regular rules end ⬆️
 ! !SECTION: Subscriptions - Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184349
+report.az##div.subs-in-social.subs-instagram
+report.az##div.subs-in-social.subs-telegram
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/subscriptions_specific.txt:7081]: Если посмотреть код, то будет видно, что можно безопасно обойтись одним, довольно однозначным, правилом
```suggestion
report.az##.subs-in-social
```
Это не ошибка, правила норм, но так лучше.

И ещё момент:

image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Other/sections/annoyances.txt [hunk 3]
```diff
@@ -3229,3 +3229,6 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 !+ PLATFORM(ios) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app#$#.hello_from_adguard_advanced_protection_ios { display: none !important; }
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184349
+report.az##div.subs-in-social.subs-instagram
+report.az##div.subs-in-social.subs-telegram
```
Comment (Alex-302) [inline AnnoyancesFilter/Other/sections/annoyances.txt:3234]: Отсюда надо удалить

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt (hunk 2 above)
Comment (Chinaski1) [reply AnnoyancesFilter/Popups/sections/subscriptions_specific.txt]: Запушил в самоке начало по якорю, ну и коммент с ссылкой на задачу тогда убираю тож

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt [hunk 4]
```diff
@@ -7076,3 +7076,5 @@ magazine.skyeng.ru##.tg-banner
 ! NOTE: Regular rules end ⬆️
 ! !SECTION: Subscriptions - Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184349
+report.az##div.subs-in-social
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/subscriptions_specific.txt]: Тэг для классов/id, когда правило вида `##` + `.`/`#`  можно не указывать.
Коммент здесь не нужнн. Правило нужно перенести вверх, тк добавил под секцией. 
```suggestion
! https://github.com/AdguardTeam/AdguardFilters/issues/184349
report.az##.subs-in-social
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**